frontend/components/FieldMedia/FieldMediaItem.jsx

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+import {h, Component} from 'preact'
+import proptypes from 'proptypes'
+import Style from 'lib/Style'
+import styles from './FieldMedia.css'
+
+export default class FieldMediaItem extends Component { 
+  static propTypes = {
+    /**
+     * A subset of the app config containing data specific to this field type.
+     */
+    config: proptypes.object,
+
+    /**
+     * The field value.
+     */
+    value: proptypes.bool
+  }
+
+  constructor(props) {
+    super(props)
+  }
+
+  getSource () {
+    const {config, isList, value} = this.props
+    const cdn = config ? config.cdn : null
+
+    if (!value) return null
+
+    const styleFile = new Style(styles, 'file')
+      .addIf('file-list', isList)
+
+    // Get the image path if applicable
+    if (value.mimetype && value.mimetype.indexOf('image') > -1) {
+      
+      let src = value._previewData ? value._previewData : value.url || value.path
+
+      if (value.path && cdn && cdn.publicUrl) {
+        src = `${cdn.publicUrl}/${value.path}?width=80`
+      }
+
+      return (
+        <div class={styles.image}>
+          <img src={src} />
+        </div>
+      )
+    } else {
```

Comment:
This `else` is redundant, I believe, as you can just move the `return` to the top level?